files/build_templates/sonic_debian_extension.j2

```diff
@@ -643,6 +643,12 @@ MLNX_PLATFORM_COMMON_PY2_WHEEL_NAME=$(basename {{mlnx_platform_api_py2_wheel_pat
 sudo cp {{mlnx_platform_api_py2_wheel_path}} $FILESYSTEM_ROOT/$MLNX_PLATFORM_COMMON_PY2_WHEEL_NAME
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip2 install $MLNX_PLATFORM_COMMON_PY2_WHEEL_NAME
 sudo rm -rf $FILESYSTEM_ROOT/$MLNX_PLATFORM_COMMON_PY2_WHEEL_NAME
+
+# Install mlnx-sonic-platform-common Python 3 package
+MLNX_PLATFORM_COMMON_PY3_WHEEL_NAME=$(basename {{mlnx_platform_api_py3_wheel_path}})
```

Comment:
Nomenclature nit-pick: The vendor-implemented API package is simply called "sonic-platform", not "sonic-platform-common". The base API package is called "sonic-platform-common". I suggest you change "mlnx-sonic-platform-common" to "mlnx-sonic-platform" and "MLNX_PLATFORM_COMMON_PY3_WHEEL_NAME" to "MLNX_SONIC_PLATFORM_PY3_WHEEL_NAME". Also suggest doing the same for the Python 2 package above.